### PR TITLE
add, edit supplies; logout fix

### PR DESCRIPTION
**Pull request by shackyshell**
Issue #38

### CODE REVIEW

File: mobile/components/ButtonsRow.js [hunk 1]
```diff
@@ -9,19 +12,33 @@ export default class ButtonsRow extends React.Component {
...
+                {/*<Button style={styles.button} onPress={() => {*/}
+                {/*signIn();*/}
+                {/*}} title={"zaloguj"}*/}
+                {/*color={"#098584"}/>*/}
+                {/*<Button style={styles.button} onPress={() => {*/}
+                {/*clearSession()*/}
+                {/*.then((res) => res && this.props.onPressLogout())*/}
+
+                {/*}} title={"Wyloguj"}*/}
+                {/*color={"#098584"}/>*/}
```
Comment (MarekPokropinski) [inline mobile/components/ButtonsRow.js]: Are these comments necessary?

File: mobile/services/AuthService.js [hunk 2]
```diff
@@ -73,16 +57,38 @@ export const signIn = async () => {
...
+    //                 'Content-Type': 'application/json',
+    //             },
+    //
+    //             body: JSON.stringify({username: login, password: password})
+    //         }
+    //     ).then(async (res) => {
+    //         console.log("res ", res)
+    //         await SecureStore.setItemAsync('secure_token', res.json().token)
+    //     }
+    // ).error((err) => console.log("błąd: ", err))
```
Comment (MarekPokropinski) [inline mobile/services/AuthService.js]: more comments

File: mobile/screens/SuppliesScreen.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                        }
+            >
+
+                {this.state.results.map((supply, index) => {
+                    return <ListItem
+                        style={styles.listItem}
+                        key={index}
+                        leftAvatar={{source: {uri: 'https://via.placeholder.com/150'}}}
+                        title={supply.name}
+                        subtitle={supply.description}
```
Comment (MarekPokropinski) [inline mobile/screens/SuppliesScreen.js]: Descriptions can be long, maybe to long to put them on the list. Maybe if description is too long only display beginning of it?

File: mobile/screens/SupplyAdd.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                                fadeDuration={0}
+                                style={styles.image}
+                            />
+                        </View>
+                        <Input inputStyle={styles.optionsTitleText}
+                               onChange={(e) => {
+                                   this.setState({supply: {...supply, name: e.nativeEvent.text}})
+                               }}
+                               value={supply.name}
+                               label={"Nazwa"}
```
Comment (MarekPokropinski) [inline mobile/screens/SupplyAdd.js:54]: I think we want to use English as language for our app.

**Review by MarekPokropinski — DISMISSED** [review]

File: mobile/screens/SupplyAdd.js (hunk 4 above)
Comment (shackyshell) [reply mobile/screens/SupplyAdd.js:54]: task for translations was added.

**Pull request merged**